partiql-types/src/lib.rs

```diff
@@ -742,6 +742,12 @@ pub enum Static {
     // TODO Add BitString, ByteString, Blob, Clob, and Graph types
 }
 
+pub enum StaticCategory<'a> {
+    Tuple(),
```

Comment:
`Struct`?

Reply:
I think we probably actually want this to represent the 'category' of tuple types. Right now, that's just `Struct`, but may include, e.g., `Row` in the future. 

Similar to how it represents both `Bag` and `List` with `Sequence`.